app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,105 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
+
+  @@object_after_convert = {}
+
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
+  end
+
+  # GET /posts/1
+  # GET /posts/1.json
+  def show
+  end
+
+  # GET /posts/new
+  def new
+    @post = Post.new
+  end
+
+  # GET /posts/1/edit
+  def edit
+  end
+
+  # POST /posts
+  # POST /posts.json
+  def create
+    
+    if post_params[:images].blank?
+      flash[:error] = "Please, choose image"
+      return
+    end
+    # ! Need refector code here
+    convert_to_url_cloudinary(post_params[:images].original_filename)
+
+    @url_image = @@object_after_convert[:landscape]['url']
```

Comment:
using class variable `@@object_after_convert` in this case is not appropriate
method `convert_to_url_cloudinary` should return an url, then 
`@url_image = convert_to_url_cloudinary(post_params[:images].original_filename)` is enough